src/modules/flow/filter-repeated/filter-repeated.c

```diff
@@ -144,12 +144,12 @@ error_filter(struct sol_flow_node *node, void *data, uint16_t port, uint16_t con
     free(mdata->msg);
     mdata->msg = strdup(in_value);
     if (!mdata->msg) {
-        sol_flow_send_error_packet(node, errno, sol_util_strerrora(errno));
+        sol_flow_send_error_packet(node, errno, "%s", sol_util_strerrora(errno));
```

Comment:
this should be pretty common usage, what if we add a static inline to do this one, just give it the errno and get it formatted and send the packet.